projects/storefrontapp/src/app/spartacus/features/cdc-feature.module.ts

```diff
@@ -27,6 +30,29 @@ import { CmsConfig, provideConfig } from '@spartacus/core';
         },
       },
     }),
+    provideConfig(<CmsConfig>{
+      featureModules: {
+        [USER_PROFILE_FEATURE]: {
+          module: () =>
+            import('@spartacus/cdc/core').then((m) => m.CDCUserProfileModule),
+        },
+      },
+    }),
+    provideConfig(<CmsConfig>{
+      cmsComponents: {
+        ReturningCustomerLoginComponent: {
+          component: LoginFormComponent,
+          guards: [NotAuthGuard],
+          providers: [
+            {
+              provide: LoginFormComponentService,
+              useClass: CdcLoginComponentService,
+              deps: [AuthService, GlobalMessageService, WindowRef, CdcJsService],
+            },
+          ],
+        },
+      },
+    }),
```

Comment:
Let's change it to:
```ts
provideConfig(<CmsConfig>{
      featureModules: {
        [USER_ACCOUNT_FEATURE]: {
          module: () =>
            import('@spartacus/cdc/???').then((m) => m.CDCUserAccountModule),
        },
      },
    }),
```

btw. we might consider renaming of the modules and paths, e.g.
```
import('@spartacus/cdc/login').then((m) => m.CDCLoginModule),
```

Reply:
As we discussed in the call, the possible alternatives for new entry points are:

1. in the root folder (sibling to /core, /components)
```
@spartacus/cdc/core
@spartacus/cdc/components
@spartacus/cdc/login
@spartacus/cdc/register
@spartacus/cdc/update-profile (in the future)
```

2. nest inside /components/
```
@spartacus/cdc/core
@spartacus/cdc/components
@spartacus/cdc/components/login
@spartacus/cdc/components/register
@spartacus/cdc/components/update-profile (in the future)
```


3. nest inside new folder /extensions/
```
@spartacus/cdc/root
@spartacus/cdc/components
@spartacus/cdc/extensions/login
@spartacus/cdc/extensions/register
@spartacus/cdc/extensions/update-profile (in the future)
```

I have hard time choosing between 2. and 3. But I'm a little more fond of 2. (nest in /components/).

### In the future, when we implement CDC update-profile:
In the future, when we implement the CDC update-profile component, it will be no longer optimal for customers to load in production the JS logic for both screenset component and the individual extended components. Then I'll suggest to allow customers for choosing (via installation schematics) which implementation do they want: screen set (dynamic iframe driven) vs. Spartacus extended components.

Then it will make sense (although it's only a cosmetic change), to move the GigyaRaasComponent (iframe creator) from `@spartacus/cdc/components` to `@spartacus/cdc/components/screen-set`.

Reply:
Pasting here the conclusion from the offline chat, for future reference:

Lately, we realized that names of entry points `@spartacus/cdc/user-profile` and `@spartacus/cdc/user-account` could better suit our needs than entry points named `@spartacus/cdc/components/register` and `@spartacus/cdc/components/login` . That’s becasue now we need to put somewhere the Forgot Password feature - which neither fits well in `/register` nor in `/login`, but rather in `/user-profile`. Moreover in the future we’re going to add the Update Profile feature, which also fits well to `/user-profile`.

I’m aware a few months ago we changed our mind from naming entry points and modules `user-profile` and `user-account` to `register` and `login` . And I’m sorry for back-and-forth suggestions in this topic.
The addition of Forgot Password and future Update Profile helped me realize that it makes sense to have `cdc/user-profile` and `cdc/user-account` entry points (and modules).